harness/determined/cli/experiment.py

```diff
@@ -426,7 +426,16 @@ def list_experiments(args: Namespace) -> None:
     if not args.all:
         users = [authentication.must_cli_auth().get_session_user()]
 
-    r = bindings.get_GetExperiments(setup_session(args), users=users)
+    all_experiments = []
```

Comment:
we've got to test this out with a big database of experiments. e.g. 50k experiments
if it has any negative effects then we may need to add the explicit user flags to control the pagination from the CLI